### DISCUSSION TITLE
RBAC with LDAP

### DISCUSSION THREAD

**Discussion by metalshanked** [provectus/kafka-ui#3190] [Q&A] [answered]
Excited for the much awaited RBAC controls in 0.5.0 but looks like LDAP is not supported yet.

is that correct? If so, is there any other workaround to make LDAP work with RBAC?
( I understand the 0.6.x version would be months away)

Thanks!

**Comment by Haarolean** [comment, marked as answer]
Hey, thanks for reaching out.

> I understand the 0.6.x version would be months away

not really. We're leaning towards quicker and smaller releases now.

LDAP support for RBAC will be implemented really soon, within #2752, I'm almost done with it.
